pkg/deploy/output.go

```diff
@@ -50,6 +50,7 @@ type Output struct {
 	IDPSyncExecutionRoleARN       string `json:"IDPSyncExecutionRoleARN"`
 	CacheSyncFunctionName         string `json:"CacheSyncFunctionName"`
 	CLIAppClientID                string `json:"CLIAppClientID"`
+	IdentityGroupFilter           string `json:"IdentityGroupFilter"`
```

Comment:
We can be clearer that we're filtering names, rather than Group IDs here

```suggestion
	IdentityGroupNameFilter           string `json:"IdentityGroupFilter"`
```